Formula/nef.rb

```diff
@@ -0,0 +1,62 @@
+class Nef < Formula
+  desc "💊 steroids for Xcode Playgrounds"
+  homepage "https://github.com/bow-swift/nef"
+  url "https://github.com/bow-swift/nef/archive/0.5.2.tar.gz"
+  sha256 "050181c1e541e54428f8214092b17a321fb1e4f2460ae2bd8115063637244b02"
+
+  depends_on :xcode => "11.0"
+
+  def version
+    "0.5.2"
+  end
+
+  def install
+    build_project
+
+    bin.install "./bin/nef"
+    bin.install "./bin/nef-common"
+    bin.install "./bin/nefc"
+    bin.install "./bin/nef-playground"
+    bin.install "./bin/nef-markdown-page"
+    bin.install "./bin/nef-markdown"
+    bin.install "./bin/nef-jekyll-page"
+    bin.install "./bin/nef-jekyll"
+    bin.install "./bin/nef-carbon-page"
+    bin.install "./bin/nef-carbon"
+    bin.install "./bin/nef-playground-book"
+  end
+
+  def build_project
```

Comment:
Don't introduce extra functions. inline the content of `build_project` in `install`.